docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,673 @@
+---
+title: "Movement Sensor Component"
+linkTitle: "Movement Sensor"
+weight: 70
+type: "docs"
+description: "A sensor that measures location, kinematic data, or both."
+tags: ["movement sensor", "gps", "imu", "sensor", "components"]
+icon: "/components/img/components/imu.svg"
+no_list: true
+aliases:
+    - /components/movement-sensor/
+# SME: Rand
+---
+
+A movement sensor component is a sensor that gives data on where a robot is and how fast it is moving.
+Examples of movement sensors include global positioning systems (GPS), inertial measurement units (IMUs), accelerometers and gyroscopes.
+
+{{% alert title="Note" color="note" %}}
+
+Viam also supports generic [sensors](/components/sensor/) and [encoders](/components/encoder/).
+
+{{% /alert %}}
+
+## Configuration
+
+Viam supports several different models of GPS, IMU and accelerometer.
+Click the model names below for configuration information:
+
+Model | Supported hardware <a name="model-table"></a>
+---------- | ------------------
+[`gps-nmea`](./gps/gps-nmea/) | [NMEA-based](https://en.wikipedia.org/wiki/NMEA_0183) GPS models
+[`gps-rtk`](./gps/gps-rtk/) | [NTRIP-based](https://en.wikipedia.org/wiki/Networked_Transport_of_RTCM_via_Internet_Protocol) [RTK](https://en.wikipedia.org/wiki/Real-time_kinematic_positioning) GPS models
+[`imu-wit`](./imu/imu-wit/) | IMUs manufactured by [WitMotion](https://witmotion-sensor.com/)
+[`imu-vectornav`](./imu/imu-wit) | IMUs manufactured by [VectorNav](https://www.vectornav.com/products)
+[`accel-adxl345`](./adxl345) | The [Analog Devices ADXL345](https://www.analog.com/en/products/adxl345.html) digital accelerometer
+[`mpu6050`](./mpu6050/) | A gyroscope/accelerometer manufactured by TDK InvenSense
+[`rtk-station`](./gps/rtk-station/) | An **experimental** model that allows you to configure your own correction source. Can be linked to an RTK-ready GPS module.

```

Comment:
let's put all the ones that are experimental or MVP [here](https://docs.google.com/spreadsheets/d/1HM4vVoqOn6C_rpPEJOrnPpys4lB2CmZg90yFmgjnbXk/edit#gid=0) in draft mode